puppet/zulip_ops/manifests/prometheus_node.pp

```diff
@@ -0,0 +1,40 @@
+# @summary configures a node for monitoring with Prometheus
+#
+class zulip_ops::prometheus_node {
+  include zulip::supervisor
+
+  $version = '1.0.1'
+  zulip::sha256_tarball_to { 'node_exporter':
+    url     => "https://github.com/prometheus/node_exporter/releases/download/v${version}/node_exporter-${version}.linux-amd64.tar.gz",
+    sha256  => '3369b76cd2b0ba678b6d618deab320e565c3d93ccb5c2a0d5db51a53857768ae',
+    install => {
+      "node_exporter-${version}.linux-amd64/node_exporter" => "/usr/local/bin/node_exporter-${version}",
+    },
+  }
+  file { '/usr/local/bin/node_exporter':
+    ensure => 'link',
+    target => "/usr/local/bin/node_exporter-${version}",
+  }
+
+  group { 'prometheus':
+    ensure => present,
+    gid    => '1060',
+  }
+  user { 'prometheus':
+    ensure     => present,
+    uid        => '1060',
+    gid        => '1060',
+    shell      => '/bin/bash',
+    home       => '/',
```

Comment:
Is `/` the usual choice for this?  I'm used to `/usr/lib/prometheus` for Debian packages, but I guess this doesn't install directories like that.

Reply:
Yeah, the raw prometheus node_exporter doesn't install any directories.  I can make an empty one for it, but that seems odd.  Ubuntu seems to use `/nonexistent` for several of them; I'll switch to that for consistency, at least.